packages/SwingSet/src/devices/lib/deviceTools.js

```diff
@@ -69,6 +69,7 @@ export function buildSerializationTools(syscall, deviceName) {
     if (devnodeSlot) {
       return devnodeSlot;
     }
+    // @ts-expect-error DetailsToken not a string
     throw Error(X`unable to convert value ${val}`);
```

Comment:
See https://github.com/Agoric/agoric-sdk/pull/6538

Reply:
Btw, it is a victory of this PR that it triggered this ts error! This is the opportune time to notice whether such newly triggered diagnostics indicate a genuine problem that we previously missed.